### DISCUSSION TITLE
[Question] `nerdctl` build on github action

### DISCUSSION THREAD

**Discussion by sureshg** [containerd/nerdctl#913] [Q&A]
We are currently using `nerdctl` along with `rancher desktop` for all the local dev. So wondering if there is any official support for running `nerdctl` build on the `Github action` workflow. Even some documentation on manually settings things up would be helpful.

**Comment by sf1tzp** [comment]
Came here wondering the same thing. I suppose you would just need to install it normally, but haven't tried yet. Did you do this in your actions since you originally posted?

**Reply by sureshg** [reply]
@sf1tzp No, we've switched to using the Docker client now.
